docs/developer-docs/latest/plugins/upload.md

```diff
@@ -467,27 +472,47 @@ module.exports = {
 };
 ```
 
+:::tip
+For performance reasons, the upload plugin will only use the `uploadStream` function if it exists, otherwise it will fallback on the `upload` function.
+:::
+
 You can then publish it to make it available to the community.
 
 ### Create a local provider
 
 If you want to create your own provider without publishing it on **npm** you can follow these steps:
 
-1. Create a `./providers/strapi-provider-upload-{provider-name}` folder in your root application folder.
+1. Create a `./providers/upload-{provider-name}` folder in your root application folder.
 2. Create your provider as explained in the [documentation](#create-providers) above.
-3. Then update your `package.json` to link your `strapi-provider-upload-{provider-name}` dependency to point to the [local path](https://docs.npmjs.com/files/package.json#local-paths) of your provider.
+3. Then update your `package.json` to link your `upload-{provider-name}` dependency to point to the [local path](https://docs.npmjs.com/files/package.json#local-paths) of your provider.
 
 ```json
-//path: ./package.json
+// ./package.json
```

Comment:
```suggestion
// path ./package.json
```

I'd rather us keep the "path" to be fully explicit. This is how we build all other code examples in the documentation :-)